src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,38 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * The card is unique to each customer.
+ */
+public final class LoyaltyCard {
+    /**
+     * Number is the card's id.
```

Comment:
Possessive voice ('s) is used for people or animals in English. It would be better to say about “an ID of a card” or ”a card identifier”.